website/astro.config.ts

```diff
@@ -417,6 +381,7 @@ export default defineConfig({
 
 		server: {
 			fs: {
+				strict: false,
```

Comment:
Vite is having some issue with the DevToolbar trying to import its code and failing because of the allow list configuration, disabling `strict` works, but would appreciate if anyone was able to fix it properly in the `allow` setting